--- dlls/winecoreaudio.drv/midi.c	2008-07-26 00:42:25.000000000 +0900
+++ dlls/winecoreaudio.drv/midi.c	2011-03-03 00:09:14.000000000 +0900
@@ -116,7 +116,7 @@
     MIDIOut_NumDevs = MAX_MIDI_SYNTHS;
     MIDIOut_NumDevs += numDest;
 
-    MIDIIn_NumDevs = MIDIGetNumberOfSources();
+    MIDIIn_NumDevs = 0;// oh no... MIDIGetNumberOfSources();
 
     TRACE("MIDIOut_NumDevs %d MIDIIn_NumDevs %d\n", MIDIOut_NumDevs, MIDIIn_NumDevs);
 
